notebooks/json_blocks/sensitivity_matrix/BLOCK-T377.ipynb

```diff
@@ -0,0 +1,199 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    for idx, range in enumerate(ranges):
</pre><p>You should not use <code>range</code> as a variable name since it is a built-in python function name. This applies to the rest of the sensitivity_matrix notebooks as well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>